### PR TITLE
add e2e coverage details

### PR DESCRIPTION
**Pull request by Vacant2333**
**What type of PR is this?**
Add features e2e test coverage details
issue:https://github.com/karmada-io/karmada/issues/3685

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Hi @Vacant2333 thanks~
Is this a template, for each feature, its `E2E test coverage` and `Coverage details` will look like in the future, can you give a simple example?

**Comment by Vacant2333** [comment]
> Hi @Vacant2333 thanks~ Is this a template, for each feature, its `E2E test coverage` and `Coverage details` will look like in the future, can you give a simple example?

hi ,yes its a template now, im working on it now and add details

**Comment by XiShanYongYe-Chang** [comment]
> hi ,yes its a template now, im working on it now and add details

Ok, I get it.

**Comment by Vacant2333** [comment]
> > hi ,yes its a template now, im working on it now and add details
> 
> Ok, I get it.

I have organized the e2e tests of Karmada, but I may not have sufficient understanding of most of the features. I need further review and suggestions.

File: docs/E2E_coverage.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+### E2E coverage of features
+
+Features: https://karmada.io/docs/key-features/features/
+
+## Cross-cloud multi-cluster multi-mode management
+| Feature               | E2E test coverage | Coverage details                                                             |
+|-----------------------|-------------------|------------------------------------------------------------------------------|
+| Safe isolation        | Yes               | https://github.com/karmada-io/karmada/blob/master/test/e2e/namespace_test.go |
```
[the remark this answers is not in the document]
Comment (Vacant2333) [reply docs/E2E_coverage.md:8]: yes its a question, but more infomations are good for this, is there another way to show the files?

**Pull request closed without merging**